### ISSUE TITLE
AVT 1 - Expandable Tile: DAP violation the form control is missing an associated label

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2483]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Run DAP with Feb. 2019 ruleset on the Expandable Tile component
See screenshot below for violation:
<img width="1060" alt="Screen Shot 2019-04-18 at 3 34 21 PM" src="https://user-images.githubusercontent.com/21676914/56389730-e3e4a000-61ef-11e9-80e8-ad7d71b376a8.png">

**Comment by snidersd** [comment]
Confirmed there are no DAP violations (see screen
<img width="1324" alt="Screen Shot 2019-06-12 at 1 11 35 PM" src="https://user-images.githubusercontent.com/21676914/59375705-59fb1400-8d14-11e9-933d-9a09bf3a883c.png">
shot)
